Libraries/Share/Share.js

```diff
@@ -114,13 +114,17 @@ class Share {
           'NativeActionSheetManager is not registered on iOS, but it should be.',
         );
 
+        const processedTintColor = processColor(tintColor);
```

Comment:
Is this processing it again? It looks like tintColor was already processed on line 110

Reply:
that was silly of me :blush: